Amplify/Categories/DataStore/Model/Temporal/Temporal.swift

```diff
@@ -7,23 +7,17 @@
 
 import Foundation
 
-public struct Temporal {
+/// `Temporal` is namespace to all temporal types and basic date operations. It can
+/// not be directly instantiated.
+///
+///  Available `Temporal` `Specs`
+/// - Temporal.Date
+/// - Temporal.DateTime
+/// - Temporal.Time
+public enum Temporal {}
 
-    /// This struct is used as a namespace to all temporal types. It should
-    /// not be directly instantiated.
-    ///
-    /// - seealso: Temporal.Date
-    /// - seealso: Temporal.DateTime
-    /// - seealso: Temporal.Time
-    /// - seealso: Temporal.TimeStamp
-    private init() {}
-}
-
-/// The `TemporalSpec` protocol defines an [ISO-8601](https://www.iso.org/iso-8601-date-and-time-format.html)
```

Comment:
It wasn't removed, it was moved to the type level so it can be viewed in quick help. The documentation here on the `private init` wasn't visible anywhere. Also the `- seealso: Temporal.TimeStamp` was removed because `Temporal.TimeStamp` doesn't exist.

Reply:
Ah, I see what you mean now. Re-added the `TemporalSpec` documentation in [22f0216](https://github.com/aws-amplify/amplify-ios/pull/1760/commits/22f02161a91baab1365c3a61058f0f9e1ab53078)